pmg/static/resources/css/style.scss

```diff
@@ -936,6 +936,18 @@ body.email_alerts {
 }
 
 // committee detail page
+.committee_detail {
+  .tab-content {
+    padding:15px;
```

Comment:
can we make this ``padding-top: 15px``? Otherwise on mobile we're losing useful spaces on the sides. On desktop I'm ambivalent.